docs/spec.md

```diff
@@ -6126,13 +6202,21 @@ works for per-tensor quantization. Per-axis quantization is work in progress
 ([#1574](https://github.com/openxla/stablehlo/issues/1574)).
 
 ```python
+
+def dequantize(x):
+  return (x - zero_point(x)) * scale(x)
+
+def quantize(x, type):
+  rounded_result = round_nearest_even(x / scale(type))
+  return clamp(storage_min(type), rounded_result + zero_point(type), storage_max(type))
```

Comment:
The sharpened definitions of dequantize and quantize from the comment above enable us to reason about a specific discussion item that came up offline:
  1) Are we comfortable computing `x_storage - zero_point(x_storage)` and `x_storage_rounded + zero_point(type)` in `storage_type(x)` or do we need to upcast to e.g. `i32` first and then downcast back to the storage type?
  2) Also, now that I'm thinking about that, does `clamp(storage_min(type), x_storage_add, storage_max(type))` even make sense as it's currently written? If `type(x_storage_add) = storage_type(x)`, then clamping to minimum and maximum values of the storage type is a no-op.

Reply:
Thanks for pointing it point. Let me propose to keep the PR as is (using the conversion to  `storage_type`) and revisit the issues separately in https://github.com/openxla/stablehlo/issues/1627.

Reply:
Sounds good.